save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/SaveOrchestrator.kt

```diff
@@ -14,7 +13,6 @@ internal typealias TextResponse = ResponseEntity<String>
  * An entrypoint for spring boot for save-orchestrator
  */
 @SpringBootApplication
-@EnableWebFlux
```

Comment:
Hm, this is very strange. This annotation makes an auto-configuration of web-flux. It can break other functionality, no?

Reply:
I tried several ways to increase buffer - by adding specific flags in `application.properties`, customize webserver directly in source code, but anytime it were ignored while annotation `@EnableWebFlux` was enabled.

I thought about it, because `@EnableWebFlux`  was added for a reason, and may be without we can break something, however I didn't met any problems while execution and large logs were send without any exceptions